ipapython/certdb.py

```diff
@@ -161,11 +219,34 @@ def get_trust_chain(self, nickname):
 
         return root_nicknames
 
-    def import_pkcs12(self, pkcs12_filename, db_password_filename,
-                      pkcs12_passwd=None):
+    def export_pkcs12(self, nickname, pkcs12_filename, pkcs12_passwd=None):
+        args = [PK12UTIL, "-d", self.secdir,
+                "-o", pkcs12_filename,
+                "-n", nickname,
+                "-k", self.pwd_file]
+        pkcs12_password_file = None
+        if pkcs12_passwd is not None:
+            pkcs12_password_file = ipautil.write_tmp_file(pkcs12_passwd + '\n')
+            args = args + ["-w", pkcs12_password_file.name]
```

Comment:
I know, I flip flopped alot around hat to use, and I settled on this for readability.